docs/OpenCart.md

```diff
@@ -0,0 +1,184 @@
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--20--debug-mode-enable.png) 
+
+You can now find the debug output in the error log in menu "System -> Maintenence -> Error Logs".
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--21--error-logs.png) 
+
+*Please make sure to disable it after debugging finished or otherwise it will fill up your error logs.*
+
+
+
+**Example Error**:   
+> 2022-05-24 21:10:50 ERROR Error during POST to https://btcpay.example.com/api/v1/stores/4kD5bvAF5j8DokHqAzxb6MFDV4ikabcdefghijklm/invoices. Got response (401): {&quot;code&quot;:&quot;unauthenticated&quot;,&quot;message&quot;:&quot;Authentication is required for accessing this endpoint&quot;}
+
+- This means there is some authentication error. Likely your api key does not have permission create invoices for that store. Make sure you gave the api key the right permissions and you give it to the right store and also entered that in OpenCart payment configuration form.
+
+- Another reason could be that you use a legacy api key. The legacy api keys are located in store settings -> Access Tokens. But you need to create an account api key which is located in Account -> Manage Account -> tab "API Keys". See section [2.2 Create an API key and configure permissions](#22-create-an-api-key-and-configure-permissions).
+
+
+## The order states do not update although the invoice has been paid
+Please check the details of your invoice if there were any errors on sending the webhook request. Some hosting providers, firewall setups or security extensions may block POST requests to your site which lead to a http status of "403 forbidden". 
+
+You can check and verify yourself if there is something blocking requests to your site in one of these two ways:
+
+**1. Copy webhook callback URL**   
+go to your *OpenCart BTCPay extension settings* and copy the "URL" of the "Webhook Data" field. e.g. `https://YOURSTOREDOMAIN.TLD/index.php?route=extension/payment/btcpay/callback`
+
+![BTCPay OpenCart: Save OpenCart Settings form](./img/opencart/oc3--12--webhook-success.png) 
+
+**2.1 Check using a command line (Linux or MacOS):**   
+
+```
+curl -vX POST -H "Content-Type: application/json" \
+    -d '{"data": "test"}' WEBHOOK_CALLBACK_URL
+```
+(replace `WEBHOOK_CALLBACK_URL` with the one copied above)
+
+Result:
+
+```
+.... snip ....
+* upload completely sent off: 16 out of 16 bytes
+< HTTP/1.1 403 Forbidden
+< access-control-allow-origin: *
+< Content-Type: application/json; charset=UTF-8
+< X-Cloud-Trace-Context: 4f07d5b2e5c2f05949d04421a8e2dd6a
+< Date: Thu, 17 Feb 2022 10:06:50 GMT
+< Server: Google Frontend
+< Content-Length: 26
+```
+
+If you see that line "HTTP/1.1 403 Forbidden" or "HTTP/2 403" then something is blocking data sent to your OpenCart site. You should ask your hosting provider or make sure no firewall or security extension is blocking the requests.
```

Comment:
```suggestion
If you see that line "HTTP/1.1 403 Forbidden" or "HTTP/2 403" something is blocking data sent to your OpenCart site. It would be best to ask your hosting provider or make sure no firewall or security extension is blocking the requests.
```